Preliminary DanubeTech implementation report.

peacekeeper opened the pull request:
https://github.com/danubetech/verifiable-credentials-java

This is an incomplete implementation with still many failures.
It shows how both Linked Data Proofs and JWT can be supported.

Report was created with this branch of the test suite:
https://github.com/uport-project/vc-test-suite/tree/jwt-support

peacekeeper committed:
Preliminary DanubeTech implementation report.

https://github.com/danubetech/verifiable-credentials-java

This is an incomplete implementation with still many failures.
It shows how both Linked Data Proofs and JWT can be supported.

Report was created with this branch of the test suite:
https://github.com/uport-project/vc-test-suite/tree/jwt-support

TallTed commented:
@peacekeeper - Noting the branch doesn't note the version/revision of the tests, which is important to knowing whether the tests need to be re-run.  I'm guessing that you ran with this version (https://github.com/uport-project/vc-test-suite/tree/ca11ce4f64e55b8a62fcd5e0f02c0ef90bddcec8) upon the switch from `iat` to `nbf` (https://github.com/uport-project/vc-test-suite/commit/ca11ce4f64e55b8a62fcd5e0f02c0ef90bddcec8)

peacekeeper commented:
@TallTed you are correct, that's the version I used.

peacekeeper committed:
Update DanubeTech report with support for JWT Verifiable Presentations.

Based on https://github.com/danubetech/verifiable-credentials-java/commit/aaa1b4c742134e48195e957491c0c5bf5121378f.
